cluster/ubuntu/util.sh

```diff
@@ -182,6 +191,11 @@ function verify-minion(){
       ssh $SSH_OPTS "$1" "pgrep -f $daemon" >/dev/null 2>&1 || {
         printf "."
         validated="1"
+        ((try_count=try_count+1))
+        if [[ ${try_count} -gt ${max_try_count} ]]; then
+          printf "\nWarning: Process \"${daemon}\" status check timeout, please check manually.\n"
```

Comment:
Will it change to `\nWarning: Process \"${daemon}\" failed to run on ${MACHINE_IP}, please check.\n`